bobweb/bob/utils_common.py

```diff
@@ -267,8 +267,12 @@ def min_max_normalize(value_or_iterable: Decimal | int | List[Decimal | int] | L
     :return:
     """
 
-    def normalization_function(x) -> Decimal:
-        return Decimal((x - old_min) * (new_max - new_min)) / Decimal((old_max - old_min)) + new_min
+    def normalization_function(x: int | Decimal) -> Decimal:
+        dividend = Decimal((x - old_min) * (new_max - new_min))
+        divisor = Decimal((old_max - old_min))
+        if divisor.is_zero():
+            return Decimal(0)
```

Comment:
What about case where old_max = 2 and old_min = 2? Maybe instead of return zero it could be return old_max (=old_min)

Reply:
I've been thinking about his and neither 0 or old_max is correct return value from this function in the sense, that they might not be in the value range specified by the new_min and new_max. I think this actually should just raise DivisionByZero exception which should then be handled in the electricity price bar graph creator.